library/std/src/sys/uefi/common.rs

```diff
@@ -0,0 +1,515 @@
+use r_efi::efi::{EventNotify, Guid, Tpl};
+
+use crate::alloc::{AllocError, Allocator, Global, Layout};
+use crate::ffi::{OsStr, OsString};
+use crate::io::{self, const_io_error};
+use crate::mem::{self, MaybeUninit};
+use crate::ops::{Deref, DerefMut};
+use crate::os::uefi;
+use crate::os::uefi::ffi::{OsStrExt, OsStringExt};
+use crate::ptr::{NonNull, Unique};
+
+pub(crate) const BOOT_SERVICES_ERROR: io::Error =
+    const_io_error!(io::ErrorKind::Other, "failed to acquire boot services",);
+pub(crate) const RUNTIME_SERVICES_ERROR: io::Error =
+    const_io_error!(io::ErrorKind::Other, "failed to acquire runtime services",);
+pub(crate) const SYSTEM_TABLE_ERROR: io::Error =
+    const_io_error!(io::ErrorKind::Other, "failed to acquire system table",);
+pub(crate) const SYSTEM_HANDLE_ERROR: io::Error =
+    const_io_error!(io::ErrorKind::Other, "failed to acquire system handle",);
+pub(crate) const VARIABLE_BOX_ERROR: io::Error =
+    const_io_error!(io::ErrorKind::OutOfMemory, "failed to allocate space for an UEFI structure",);
```

Comment:
```suggestion
    const_io_error!(io::ErrorKind::OutOfMemory, "failed to allocate space for a UEFI structure",);
```

Reply:
Removed this constant as suggested in an earlier Conversation.